client/extensions/woocommerce/state/sites/settings/stripe-connect-account/actions.js

```diff
@@ -206,7 +210,8 @@ export const deauthorizeAccount = ( siteId, successAction = null, failureAction
 		} );
 };
 
-function deauthorizeSuccess( siteId ) {
+// eslint-disable-next-line no-unused-vars
+function deauthorizeSuccess( siteId, action, data ) {
```

Comment:
Do we need to include `action` and `data` here if we are not using either of them? It should be fine to leave them off so you don't need the eslint line.

Reply:
I liked keeping them consistent with the others - is it OK to keep it that way?

Reply:
Yeah it should be fine.

Reply:
Did make one fix in this regard in https://github.com/Automattic/wp-calypso/pull/19300/commits/3937df0266fb81e6a4bc2e4b44d35fde0ed8f572